#Details of service data types
bool request
---
bool response 